tests/e2e/autoscaling/00-assert.yaml

```diff
@@ -8,6 +8,7 @@ metadata:
   name: example
 status:
   phase: Running
+  restarts: 0
```

Comment:
I don't have a better solution though.

Reply:
AFAIK we don't _need_ to specify the restart count in the status. I thought it'd be worthwhile though, because in general the VM should not need to restart during e2e tests.

Reply:
One might argue that each test should have a single concern, and we test restart counts in the restart test.

Reply:
I think ultimately this isn't "testing restart counts" as a feature, but instead _using_ restart counts to improve the coverage of the other tests (e.g., we'd want live migration to fail if it causes the VM to restart).